urwid/tests/test_event_loops.py

```diff
@@ -201,14 +201,14 @@ def test_error(self):
             evl.alarm(0.5, lambda: 1 / 0)  # Simulate error in event loop
             self.assertRaises(ZeroDivisionError, evl.run)
 
-        async def test_coroutine_error(self):
+        def test_coroutine_error(self):
             evl = self.evl
 
             async def error_coro():
```

Comment:
this syntax is not supported on python2.
Since sunset of python2, no reason to add magic with compile